ui/app/pages/swaps/awaiting-swap/__snapshots__/awaiting-swap.test.js.snap

```diff
@@ -0,0 +1,958 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AwaitingSwap renders the component with initial props 1`] = `
+<div>
+  <div
+    class="awaiting-swap"
+  >
+    <div
+      class="awaiting-swap__content"
+    >
+      <div
+        style="z-index: 0;"
+      >
+        <svg
```

Comment:
this is one where we should break apart this snapshot, hopefully in such a way that we don't end up covering the SVG / animation with a snapshot. Polygons. Polygons everywhere.

Reply:
Haha yes! This is a great example when taking a snapshot of the whole component is not easily readable.

Reply:
Done.